css-cascade/specificity.css

```diff
@@ -0,0 +1,19 @@
+* {
```

Comment:
These selectors should be done in reverse order as indicated in the instructions